Nero/src/NeroFatJets.cpp

```diff
@@ -4,67 +4,180 @@
 
 NeroFatJets::NeroFatJets() : 
         NeroCollection(),
-        BareFatJets()
+        BareFatJets(),
+        mRunJEC(false),
+        mMinId("loose"),
+        jetRadius(0.8),
+        mMCJetCorrector(0),
+        mDataJetCorrector(0)
 {
 }
 
 NeroFatJets::~NeroFatJets(){
+  if (mMCJetCorrector)   delete mMCJetCorrector;
+  if (mDataJetCorrector) delete mDataJetCorrector;
```

Comment:
you don't need the if ..., you may want to use this template function to delete:
https://github.com/MiT-HEP/NeroProducer/blob/master/Core/interface/BareFunctions.hpp#L21
so it will also be set to NULL